libdnf5/module/module_profile.cpp

```diff
@@ -64,16 +64,20 @@ bool ModuleProfile::is_default() const {
     if (!profile) {
         return {};
     }
-    return modulemd_profile_is_default(profile);
+    return is_default_profile;
```

Comment:
I believe that following code could be removed (lane 64 - 66)
```
    if (!profile) {
        return {};
    }
```

Reply:
Be honest I don't like for a method that return `bool` the following construct - `return {}`; I am not sure whether it returns `true`, `false` or uninitialized value (random).

Reply:
It was a remnant from dnf4. I am not sure why it was used like that there, as it probably makes sense only for more complicated types. But anyway, you are right it can be removed.